src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/SemanticTokensCache.cs

```diff
@@ -107,6 +109,11 @@ internal sealed class SemanticTokensCache
             if (!_cache.TryGetValue(uri, out var documentCache) ||
                 !documentCache.TryGetValue(semanticVersion, out var lineToTokensDict))
             {
+                if (logger is not null)
+                {
+                    logger.LogInformation($"No cached results found for range: {requestedRange}");
```

Comment:
Ditto with `logger?.LogInformation` but also will this get logged every time a user types a character in the doc due to the document version changing? Or is that call path somewhere else?

Reply:
Ahhh you're right - I'll remove this logging in that case

Reply:
Or actually, it's just logging at the information level, so maybe we can keep it in? We've previously logged at the information level for all semantic tokens calls anyway in the [semantic tokens endpoint](https://github.com/dotnet/razor-tooling/blob/ede2dc4b2ce2d104c6fcd5994d7ff33b20c48651/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/RazorSemanticTokensEndpoint.cs#L48).

Reply:
It's technically a highlllly hit call path so I hesitate to add more logging at this layer especially given that we're concating the string. That being said if this log will significantly aid in our ability to understand when /how things go wrong then I could be convinced